dashboard/controllers/filesController.go

```diff
@@ -0,0 +1,76 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+)
+
+const (
```

Comment:
IDK but the indentation seems to be too big in this file.

Reply:
Spittin facts